article/app/views/liveblog/pinnedPost.scala.html

```diff
@@ -0,0 +1,28 @@
+@import org.joda.time.DateTimeZone
```

Comment:
rename file and references to pinned block

Reply:
addresed in ab/pinned-block-rename